pages/terra/policy.mdx

```diff
@@ -46,69 +58,171 @@ description: "One of Terra's main focuses is package quality. Users should be ab
+</div>
+
+Packages SHOULD follow one of the version formats below:
+
+```rpmspec
+## (a)
+Version:  %{latest_stable_version}
+## (b)
+Version:  %{latest_stable_version}^%{commit_date}git%{shortcommit}
+## (c)
+Version:  %{unreleased_version}~%{commit_date}git%{shortcommit}
+```
+
+`git` SHOULD be replaced with the correct _VCS identifier_ for **(b)** and **(c)** as listed on the right.
+
+**(b)** and **(c)** MUST be used with `labels { nightly = 1 }{:hcl}` in `anda.hcl` unless there is no `update.rhai`,
+as they rely on the assumption that there is 1 `%shortcommit` for each `%commit_date` (and vice versa).
+
+`%unreleased_version` refers to the expected version number for the next stable (or a more stable _tag_ than `nightly`) release,
+while `%latest_stable_version` refers to the current latest released version number, preferably stable.
+
+**(c)** SHOULD be preferred over **(b)** if possible, since having a `%unreleased_version` can indicate to users
+breaking changes / new features depending on the versioning system (usually SemVer).
+
+If upstream does not provide a version, use **(c)** and treat `%unreleased_version` as `0`.
+
+### Release Field
+
+The `Release:{:rpmspec}` preamble MUST NOT contain information regarding upstream versioning,
+including but not limited to commit dates, commit SHAs, software versions, _tags_, etc.
+Without approval from the team, packages MUST use the format `Release: 1%?dist{:rpmspec}` (or `1%{?dist}{:rpmspec}`).
+
+The release number SHOULD be incremented if changes are made to the package, unless the `Version:{:rpmspec}` changes.
+
+### Development and Shared Libraries
+
+Shared library packages SHOULD be suffixed with `-libs`.
+These packages SHOULD NOT contain anything **except**:
+
+```rpmspec
+%_libdir/*.so.*
+%doc …
+%license …
+```
+
+Development library packages SHOULD be suffixed with `-devel`.
+These packages SHOULD NOT contain anything **except**:
+
+```rpmspec
+%doc …
+%license …
+%_includedir/*
+# ^ source files, depending on the language, could be in other locations
+%_libdir/*.so
+%_libdir/pkgconfig/%{name}.pc
+# and other development files (.vapi, .typelib, .gir, …)
+```
+
+### Naming
 
 Prefix/Suffix the packages in the following order
-(that means nightly fonts should end with `-nightly-fonts`)
+(that means nightly fonts should end with `-fonts%.nightly`)
 
-- Nightly packages must end with `-nightly`.
-- Fonts must end with `-fonts`.
-- Libraries should preferably start with `lib`.
-- Packages with names that would conflict with Fedora's packages should be prefixed with `terra-`. This is usually used for packages that are severely outdated or poorly maintained in Fedora.
+- Fonts MUST be suffixed with `-fonts`.
+- Shared libraries SHOULD be either suffixed with `-libs` or prefixed with `lib`.
+- Development libraries SHOULD be suffixed with `-devel`.
+- If necessary, suffix `-bash-completion` (or other shells) and `-doc`.
+- Postfix a _tag_ if necessary (e.g. `%.nightly`).
```

Comment:
We should really come up with a better notation for tags, because I fear using `%` may cause some footguns down the line due to the fact that it's an RPM macro notator

Reply:
then it's by design a bug that rpm should fix 🚎